docs/reference/errors-and-warnings/NU1803.md

```diff
@@ -21,5 +21,16 @@ You may need to do one of the following:
 
 - Correct the specified url. You may have mistyped the source url. Change the url to explicitly request the `HTTPS` version of the source.
 - Work with the owner of the source and ask them to migrate to `HTTPS`.
+- If you are absolutely certain the 'HTTP' server you are connecting to is trustworthy, you may opt out of this warning by setting `allowInsecureConnections` to `true` for this package source, in [Package source sections](..\nuget-config-file.md#package-source-sections) of your NuGet.Config file. (Supported in NuGet 6.8+)
```

Comment:
https doesn't protect against servers that are untrustworthy, they only protect against a NETWORK that is untrustworthy. If a hacker gains control of the server, the https certificate isn't going to help protect anything. But if you go to the command line and do `tracert <hostname>`, for example `tracert api.nuget.org`, https tries to protect against any one of those hops from being malicious and potentially delivering content that is different to what the server itself wanted you to see.